.travis.yml

```diff
@@ -2,22 +2,10 @@
 language: ruby
 
 rvm:
-  - 1.9.3
-  - 2.1.0
   - 2.3.1
```

Comment:
The pc1 aio packaging still ships with ruby 2.1.9
```shell
/opt/puppetlabs/puppet/bin/ruby --version
ruby 2.1.9p490 (2016-03-30 revision 54437) [x86_64-linux]
```

Unless there are plans from puppet to change that, I think we should stick with 2.1.9 and continue to test against that.
If we want to do 2.3.1 also, that's probably OK.